components/District.tsx

```diff
@@ -1,13 +1,18 @@
 type props = {
+  province: number;
   shape: string;
 };
 
-export default function District({ shape }: props) {
+const colors: string[] = Array(7)
+  .fill('#')
+  .map((c) => c + Math.random().toString(16).substring(2,8));
```

Comment:
This could be utility function but I would rather go with RBG or HSL color generation so that we can adjust color.